pkg/hostagent/hostagent.go

```diff
@@ -173,14 +173,14 @@ func New(instName string, stdout io.Writer, signalCh chan os.Signal, opts ...Opt
 
 	limaDriver := driverutil.CreateTargetDriverInstance(&driver.BaseDriver{
 		Instance:     inst,
-		Yaml:         y,
+		Yaml:         instConfig,
```

Comment:
I think this field should also be renamed:

```suggestion
		InstConfig:   instConfig,
```

Reply:
Renamed.